contracts/src/v0.7/dev/KeeperRegistrarDev.sol

```diff
@@ -38,35 +38,34 @@ contract KeeperRegistrarDev is TypeAndVersionInterface, ConfirmedOwner {
 
   bytes4 private constant REGISTER_REQUEST_SELECTOR = this.register.selector;
 
-  uint256 private s_minLINKJuels;
   mapping(bytes32 => PendingRequest) private s_pendingRequests;
 
   LinkTokenInterface public immutable LINK;
 
   /**
    * @notice versions:
    * - KeeperRegistrar 1.1.0: Add functionality for sender allowlist in auto approve
+   *                        : Remove rate limit and add max allowed for auto approve
    * - KeeperRegistrar 1.0.0: initial release
    */
   string public constant override typeAndVersion = "KeeperRegistrar 1.1.0";
 
-  struct AutoApprovedConfig {
-    AutoApproveType configType;
-    uint16 allowedPerWindow;
-    uint32 windowSizeInBlocks;
-    uint64 windowStart;
-    uint16 approvedInCurrentWindow;
+  struct Config {
```

Comment:
Compacting all config into same struct, generalising it from AutoApprovedConfig to Config

Reply:
I think the rename is a good call :+1: